packages/sanity/src/core/form/inputs/PortableText/Compositor.tsx

```diff
@@ -437,6 +447,9 @@ export function Compositor(props: Omit<InputProps, 'schemaType' | 'arrayFunction
       onItemOpen,
       onPaste,
       path,
+      props.elementProps,
+      props.onUpload,
+      props.resolveUploader,
       rangeDecorations,
```

Comment:
Nitpick, could we destruct these as well, as there's a bit of inconsistency in how they are listed with the other deps?

Reply:
Updated.